virttest/utils_kernel_module.py

```diff
@@ -132,22 +177,21 @@ def restore(self):
         +-------------------+-+-+-+-+
         """
 
-        if self._loaded_config is not None:
-            cmds = []
-            # TODO: Handle cases were module cannot be removed
-            cmds.append('modprobe -r --remove-dependencies %s' % self._module_name)
+        if self._get_serialized_config() != self._config_backup:
```

Comment:
Looks `if self.current_config != self._config_backup:` is better.

Reply:
Yes, I added its property, corrected it.